code/game/machinery/telecomms/telecomunications.dm

```diff
@@ -48,8 +48,7 @@ GLOBAL_LIST_EMPTY(telecomms_list)
 		signal.data["slow"] = netlag
 
 	// Loop through all linked machines and send the signal or copy.
-	for(var/m_typeless in links) // Yogs -- God bless typeless for-loops
-		var/obj/machinery/telecomms/machine = m_typeless
+	for(var/obj/machinery/telecomms/machine in links) // Yogs -- God bless typeless for-loops -- why do you need to use those here

```

Comment:
they're faster.

Reply:
They are indeed faster. Check CONTRIBUTING.md for details.

Reply:
they may be faster but what is non tcomms machine is linked? The way i understand it there is nothing to prevent that and would cause an error with having wrong type assignment.